test.py

```diff
@@ -2,41 +2,238 @@
 
 import logging
 import unittest
+from unittest.mock import Mock, mock_open, patch
```

Comment:
@TraXIcoN I think, the `mock_open` is not being used anywhere in the test suites. Could you verify it once and remove that if not being used.